tracing-undertow/build.gradle

```diff
@@ -24,6 +24,7 @@ dependencies {
     implementation project(':tracing-api')
     implementation 'com.google.guava:guava'
     implementation 'com.palantir.safe-logging:preconditions'
+    implementation "org.slf4j:slf4j-api"
```

Comment:
This shouldn't be necessary after the safe-logger migration, just in `testImplementation`, right?

Reply:
Yep, after merging in develop this disappeared